system/jlib/jscm.hpp

```diff
@@ -94,9 +93,11 @@ template <class CLASS> class Shared
 public:
     inline Shared()                              { ptr = NULL; }
     inline Shared(CLASS * _ptr, bool owned)      { ptr = _ptr; if (!owned && _ptr) _ptr->Link(); }
-    inline Shared(const Shared<CLASS> & other)   { ptr = other.getLink(); }
+    inline Shared(const Shared & other)          { ptr = other.getLink(); }
+#if defined(__cplusplus) and __cplusplus >= 201100
```

Comment:
It wasn't the check for the version I was quibbling about, it was the defined(__cplusplus)